modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -0,0 +1,110 @@
+
+.Procedure
+
+. Configure the storage class, persistent volume claim, persistent volume, and image registry configuration in the appropriate `PolicyGenTemplate` CR. For example, to configure an individual site, use the following YAML:
++
+[source,yaml]
+----
+sourceFiles:
+   # storage class
+   - fileName: StorageClass.yaml
+     policyName: "sc-for-image-registry"
+     metadata:
+       name: image-registry-sc
+       annotations:
+         ran.openshift.io/ztp-deploy-wave: "100" <1>
+  # persistent volume claim
+   - fileName: StoragePVC.yaml
+    policyName: "pvc-for-image-registry"
+    metadata:
+      name: image-registry-pvc
+      namespace: openshift-image-registry
+        annotations:
+          ran.openshift.io/ztp-deploy-wave: "100"  <2>
+    spec:
+      accessModes:
+        ReadWriteMany
+      resources:
+        requests:
+          storage: 100Gi
+      storageClassName: image-registry-sc
+      volumeMode: Filesystem
+  # persistent volume
+  - fileName: ImageRegistryPV.yaml <3>
+    policyName: "pv-for-image-registry"
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+  # image registry config
+  - fileName: ImageRegistryConfig.yaml <5>
+    policyName: "config-for-image-registry"
+    complianceType: musthave <5>
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100" <6>
+    spec:
+      storage:
+        pvc:
+          claim: "image-registry-pvc"
+----
+<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is appropriate for an individual site.  ZTP deploy waves are used to order how policies are applied to the spoke cluster. All policies created by `PolicyGen` have a ztp deploy wave by default.
+<2> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
+<4> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+<5> Configure registry to point to the PVC created above.
+<6> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+
+.Verification
+
+. Check that the CRD `Config` of the group `imageregistry.operator.openshift.io` instance is not reporting any errors.
+
+. A few minutes after the installation process is complete, you should see the `PersistentVolumeClaim` filling up.
+
+. Check that the `registry*` pod is up correctly located under the `openshift-image-registry` namespace.
+
+From inside the node:
+
+. Verify successful login to the registry with `podman`:
++
+[source,terminal]
+----
+  oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
```

Comment:
This is actually two commands, and they are both missing the $ symbol. What about:

```suggestion
[source,terminal]
----
$ oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
----
+
[source,terminal]
----
podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
----
```

Reply:
Done.